runtime.md

```diff
@@ -100,9 +100,10 @@ This operation MUST [generate an error](#errors) if it is not provided a path to
 If the ID provided is not unique across all containers within the scope of the runtime, or is not valid in any other way, the implementation MUST [generate an error](#errors) and a new container MUST NOT be created.
 This operation MUST create a new container.
 
-All of the properties configured in [`config.json`](config.md) except for [`process`](config.md#process) MUST be applied.
+All of the properties configured in [`config.json`](config.md) except for [`process`](config.md#process) and [Platform](spec.md#platforms)-specific configuration beyond the target platform MUST be applied.
```

Comment:
I feel like this is the idea that [the extensivility section][1] was trying to convey,and which I was trying to tighten up in #680.  Instead of attempting to repeat the condition here, I would rather update that section to cover the non-ignore-ability of properties defined by this spec for that target platform (even ones like `process.rlimits`, which lie outside of `linux`).  Text here that *references* that section would be great, though.  Maybe something like:

    All of the [recognized properties](config.md#extensibility) configured in [`config.json`](config.md) except for [`process`](config.md#process) MUST be applied.

with wording in the extensibility section and/or glossary to more formally define "recognized properties"?

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/config.md#extensibility

Reply:
@wking I don't think the extensibility section conveys this. In my understanding, `unknown property` means the property is not defined in spec, not the property that runtime doesn't support (if we have different understanding, maybe glossary for `unknown property` is needed?).

Reply:
> I don't think the extensibility section conveys this.

Not yet (#680 is me trying to tighten part of it down).  I'm staying that the extensibility section is where we already attempt to cover this sort of thing, not that it already covers the hole you've identified here.

I'd rather tighten up the extensibility section to cover all of the “which properties can the runtime ignore?” conditions, because if it covers a subset, and `runtime.md` adds a few more, then config authors might read the extensibility section, think they understand what the runtime can ignore, and then be surprised when it also ignores something that was added in `runtime.md`.  It's easier to understand if we collect all the conditions in one place.